app/models/ruby_method.rb

```diff
@@ -23,6 +23,10 @@ def object_constant
     body[:object_constant]
   end
 
+  def signitures
+    body[:signitures]
```

Comment:
Signatures actually has an "a", not "i". Only commenting on this one, though there's a few other spots in this PR with the same typo.

```suggestion
  def signatures
    body[:signatures]
```

Reply:
Was in the middle of fixing this but got distracted. Thanks for reminding me.